tests/MoverGroupper.test.tsx

```diff
@@ -969,4 +969,115 @@ describe("MoverGroupper", () => {
+                        id="outer-groupper"
+                        tabIndex={0}
+                        {...getTabsterAttribute({
+                            groupper: {
+                                tabbability:
+                                    Types.GroupperTabbabilities
+                                        .LimitedTrapFocus,
+                            },
+                        })}
+                    >
+                        <button>Button2</button>
+                        <div
+                            {...getTabsterAttribute({
+                                mover: {
+                                    visibilityAware:
+                                        Types.Visibilities.PartiallyVisible,
+                                },
+                            })}
+                        >
+                            <button>Button3</button>
+                            <div
+                                id="inner-groupper"
+                                {...getTabsterAttribute({
+                                    groupper: {
+                                        delegated: true,
+                                        tabbability:
+                                            Types.GroupperTabbabilities
+                                                .LimitedTrapFocus,
+                                    },
+                                })}
+                            >
+                                <button>Button4</button>
+                                <button>Button5</button>
+                            </div>
+                        </div>
+                        <button>Button6</button>
+                    </div>
+                    <button>Button7</button>
+                </div>
+            )
+        )
+            .pressTab()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button1");
+            })
+            .pressTab()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual(
+                    "Button2Button3Button4Button5Button6"
+                );
+            })
+            .pressTab()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button7");
+            })
+            .pressTab(true)
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual(
+                    "Button2Button3Button4Button5Button6"
+                );
+            })
+            .pressEnter()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button2");
+            })
+            .pressTab()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button3");
+            })
+            .pressTab()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button6");
+            })
+            .pressTab(true)
+            .activeElement((el) => {
```

Comment:
- focus button 3 (outside delegated)
- tab -> focus button 6 (outside delegated)
- shift + tab -> focus button 4 (inside delegated)

Why does tab move to the next element after the delegated groupper but shift+tab move to the first element of the delegated groupper ?

Reply:
It's mover/groupper case:

```<mover>
  <button>Button3</button>
  <groupper>
    <button>Button4</button>
    <button>Button5</button>
  </groupper>
</mover>
<button>Button6</button>```

Tab from Button3 moves you outside of the mover. Tab backwards moves you to the last element in the mover which is our groupper, inactive groupper is treated atomically and first focusable (or the groupper container if the container is focusable) is the element to be focused.

Reply:
ah yeah that makes sense from the mover's perspective